jack/pcm_jack.c

```diff
@@ -603,7 +602,11 @@ static int snd_pcm_jack_open(snd_pcm_t **pcmp, const char *name,
 	}
 
 	if (client_name == NULL) {
+#if defined(__linux__)
 		const char *pname = program_invocation_short_name;
```

Comment:
This needs GNU_SOURCE for e.g. musl. Maybe a configure check would be better or checking if _GNU_SOURCE is set (maybe in addition to __linux__).

Reply:
how about to simply replace `#if defined(__linux__)` to `#if defined(_GNU_SOURCE)` ?

Reply:
wfm